kernel/packages/shared/apis/SceneStateStorageController/SceneStateStorageController.ts

```diff
@@ -350,6 +353,28 @@ export class SceneStateStorageController extends ExposableAPI implements ISceneS
     const result = await Promise.all(promises)
     return new Map(result)
   }
+
+  private async updateProjectDetails(sceneState: SerializedSceneState, sceneName: string, sceneDescription: string, thumbnailBlob: Blob) {
+    // Deserialize the scene state
+    const deserializedSceneState: SceneStateDefinition = deserializeSceneState(sceneState)
+
+    // Convert the scene state to builder scheme format
+    let builderManifest = await toBuilderFromStateDefinitionFormat(
+      deserializedSceneState,
+      this.builderManifest,
+      this.builderApiManager
+    )
+
+    // Update the project info
+    builderManifest.project.title = sceneName
+    builderManifest.project.description = sceneDescription
+
+    // Update the manifest
+    await this.builderApiManager.updateProjectManifest(builderManifest, this.getIdentity())
+
+    // Update the thumbnail
+    await this.builderApiManager.updateProjectThumbnail(builderManifest.project.id, thumbnailBlob, this.getIdentity())
```

Comment:
This should be done in the same line, we should update the manifest and only call to the API 1 time. The builder API will save the project and manifest together, so we are making 2 calls for the same purpose

Reply:
It is done in this way because it is what the Builder API currently offers. Do you think it is worth changing the API to unify these 2 calls?

Reply:
My bad, I thought that it was updating the manifest on one hand and the project on the other. NVM!